MediaBrowser.Model/Dlna/ResolutionNormalizer.cs

```diff
@@ -13,7 +13,7 @@ public class ResolutionNormalizer
                 new ResolutionConfiguration(720, 950000),
                 new ResolutionConfiguration(1280, 2500000),
                 new ResolutionConfiguration(1920, 4000000),
-                new ResolutionConfiguration(3840, 35000000)
+                new ResolutionConfiguration(3840, 80000000)
```

Comment:
reading the code it seems that you're setting `80 mbps` as a top limit for 1080p, not for 4K

Reply:
How so? The resolution is by x-axis right

Reply:
If you play a 4K file that has a bitrate greater than 35Mb/s, it will not scale down to 1080p. Bitrate less than 35Mb/s will correctly scale down.

Reply:
So?.. I still fail to understand what issue you're fixing with this PR...

Reply:
https://github.com/jellyfin/jellyfin/issues/2093

Reply:
the issue is that some 4k files with high bitrate dont get scaled down 1080p, which causes h264 encoding in 4k (slow)

Reply:
I now get what you're changing here, but I still don't think it's the proper way of changing stuff.
Example: viewing 4K content on a 4K client, transcoding to a, say, 50 mbps from 80mbps - I don't want it to downscale.

All this "normalization" is designed badly, and juggling some hardcoded numbers feels wrong anyway...

Reply:
I know what you're saying, I said the same thing. The code is not very good(but i dont have the know how or the time to write something so involving at the moment). Majority of browsers can't play hevc so I could argue that HEVC to AVC transcode in 4K is a waste of resources, regardless weather the display supports 4K. Theres 4 ways to go around fixing this. 1) We will downscale to 1080P everytime 4K source is present that requires transcoding to AVC. 2) We will always attempt to play 4K in the web browser while transcode from HEVC to AVC. 3) We will rewrite the code 4) We will not touch the code and leave the algorithm as unpredictable as it is now. (it took me close to 2 hours to find out why some of my 4k videos get transcoded and others dont)